community/io/src/test/java/org/neo4j/graphdb/mockfs/EphemeralFileSystemAbstraction.java

```diff
@@ -1204,6 +1203,12 @@ private ByteBuffer allocate( int sizeIndex )
             }
         }
 
+        private int capacity( int sizeIndex )
+        {
+            return (sizeIndex < SIZES.length) ?
+                           SIZES[sizeIndex] : ((sizeIndex - SIZES.length + 1) * SIZES[SIZES.length - 1]);
```

Comment:
@burqen current change try to use what ever method is there for expected size calculation, lets change way how ephemeral fs allocates it buffers in 3.2 since we can make it all together much more simpler and that will not gonna require either size index either conversion between size and index. And we will not gonna have any confusion around that